packages/terra-search-field/tests/jest/SearchField.test.jsx

```diff
@@ -36,6 +36,12 @@ describe('Snapshots', () => {
 
     expect(searchField).toMatchSnapshot();
   });
+
+  it('renders a search field with an Input that contains a non-null refCallback prop', () => {
+    const searchField = shallow(<SearchField inputFieldRefCallback={(inputRef) => { inputRef.focus(); }} />, intlContexts.shallowContext);
+    expect(searchField.find('Input').props().refCallback).not.toBeFalsy();
+    expect(searchField).toMatchSnapshot();
+  });
```

Comment:
We can test the callback similarly to how it was done in [Button Group](https://github.com/cerner/terra-core/blob/f757f0918ccb7b071829726cff296f389e395466/packages/terra-button-group/tests/jest/ButtonGroup.test.jsx#L32).

Reply:
Will make the change!